ui/src/app/routing/signed-in-app-routing.tsx

```diff
@@ -328,6 +333,9 @@ export const SignedInRoutes = () => {
           routeData={{ title: 'Data Access Requirements' }}
         />
       </AppRoute>
+      <AppRoute exact path='/runtimes'>
```

Comment:
I'm glad you got the change to this file down to the same as it would be for any other new page 👍